packages/cli/internal/pkg/cli/types/context.go

```diff
@@ -21,7 +21,8 @@ type ContextInstance struct {
 }
 
 type ContextName struct {
```

Comment:
nit: Could we change struct name from ContextName to ContextDetails or something excluding Name